job_spec/INSAR_ISCE_TEST.yml

```diff
@@ -60,7 +60,7 @@ INSAR_ISCE_TEST:
         - Ref::granules
         - --secondary-scenes
         - Ref::secondary_granules
-      timeout: 10800
+      timeout: 21600
```

Comment:
According to @cmarshak ESD (and ionosphere) may 2x (or more) the job run time. This doubles the timout